media/js/firefox/australis/fx38_0_5/firstrun.js

```diff
@@ -83,14 +11,14 @@
             $document = $(document);
 
             // signup CTA opens about:accounts
-            $ctaSignup.on('click', function(e) {
+            $('#cta-signup').on('click', function(e) {
                 e.preventDefault();
 
                 Mozilla.UITour.showFirefoxAccounts();
             });
 
             // signin CTA opens hamburger menu
-            $ctaSignin.on('click', function(e) {
+            $('#cta-signin').on('click', function(e) {
```

Comment:
Any reason we cannot keep this one and the one just above as cached variables as before?